views/pages/home.ejs

```diff
@@ -13,29 +13,37 @@
 <%- include('../partials/head'); %>
     <body>
         <%- include('../partials/header'); %>
-            <main class="home">
-                <% for(i=0; i < users.length; i++){ %>
-                    <section class="account-item">
-                        <a href="/user/<%= users[i]._id; %>">
-                            <div>
-                                <% if (users[i].picture){ %>
-                                    <img src="<%= users[i].picture %>" alt="profile picture of <%= users[i].name; %>">
-                                    <% } else { %>
-                                        <span class="material-icons">account_circle</span>
-                                        <% } %>
-                            </div>
-                            <div>
-                                <p>
-                                    <%= users[i].name; %> <%= getAge(users[i].birthdate); %>
-                                </p>
-                                <p>
-                                    <%= users[i].residence; %>
-                                </p>
-                                <span class="profile">view profile</span>
-                            </div>
-                        </a>
-                    </section>
+            <main class="home"> 
+                <form method="POST" action="/filter/">
+                    <select name="gender" id="gender">
+                        <option name="Male" value="Male">Male</option>
+                        <option name="Female" value="Female">Female</option>
+                        <option name="Non-binary" value="Non-binary">Non-binary</option>
+                      </select>
+                    <input type="submit" value="Save changes">
+                </form>
+                <% for(i=0; i < users.length; i++){ { %>
+                        <section class="account-item">
+                            <a href="/user/<%= users[i]._id; %>">
+                                <div>
+                                    <% if (users[i].picture){ %>
+                                        <img src="<%= users[i].picture %>" alt="profile picture of <%= users[i].name; %>">
+                                        <% } else { %>
+                                            <span class="material-icons">account_circle</span>
+                                            <% } %>
+                                </div>
+                                <div>
+                                    <p>
+                                        <%= users[i].name; %> <%= getAge(users[i].birthdate); %>
+                                    </p>
+                                    <p>
+                                        <%= users[i].residence; %>
+                                    </p>
+                                </div>
+                            </a>
+                        </section>
```

Comment:
Kijk dit nog ff goed na tijdens het mergen

Reply:
Je bedoelt of er niks geks gebeurt met de matches.ejs die eerst de home was?